Traceback (most recent call last):
  File "c:\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\anaconda3\lib\site-packages\nbclient\client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\anaconda3\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\anaconda3\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "c:\anaconda3\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "c:\anaconda3\lib\site-packages\nbclient\client.py", line 541, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\anaconda3\lib\site-packages\nbclient\client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "c:\anaconda3\lib\site-packages\nbclient\client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
conceptos = ['hashable', 'inmutable', 'interactivo', 'interpretado', 'iterable']
definiciones = ['Si su valor "hash" nunca cambia. Pueden actuar como "clave de diccionarios".',
                'Objeto con valor fijo.',
                'Python posee un int√©rprete interactivo.',
                'Python es un lenguaje interpretado.',
                'Objeto capaz de devolver sus elementos, uno cada vez.']

str = input('Diga concepto a buscar:')

indice = conceptos.index(str)
definicion = definiciones[indice]

print(definicion)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
[1;32m<ipython-input-5-c62a4a35e936>[0m in [0;36m<module>[1;34m[0m
[0;32m      6[0m                 'Objeto capaz de devolver sus elementos, uno cada vez.']
[0;32m      7[0m [1;33m[0m[0m
[1;32m----> 8[1;33m [0mstr[0m [1;33m=[0m [0minput[0m[1;33m([0m[1;34m'Diga concepto a buscar:'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      9[0m [1;33m[0m[0m
[0;32m     10[0m [0mindice[0m [1;33m=[0m [0mconceptos[0m[1;33m.[0m[0mindex[0m[1;33m([0m[0mstr[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;32mC:\Anaconda3\lib\site-packages\ipykernel\kernelbase.py[0m in [0;36mraw_input[1;34m(self, prompt)[0m
[0;32m    856[0m         [1;32mif[0m [1;32mnot[0m [0mself[0m[1;33m.[0m[0m_allow_stdin[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0;32m    857[0m             raise StdinNotImplementedError(
[1;32m--> 858[1;33m                 [1;34m"raw_input was called, but this frontend does not support input requests."[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    859[0m             )
[0;32m    860[0m         return self._input_request(str(prompt),

[1;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.
StdinNotImplementedError: raw_input was called, but this frontend does not support input requests.

